specification/liftrneon/Neon.Postgres.Management/main.tsp

```diff
@@ -85,113 +85,74 @@ interface Organizations {
 @armResourceOperations
 @added(Neon.Postgres.Versions.v2_preview)
 interface Projects {
-  get is ArmResourceRead<Project>;
-  createOrUpdate is ArmResourceCreateOrReplaceAsync<Project>;
-  update is ArmResourcePatchAsync<Project, Models.ProjectProperties>;
-  delete is ArmResourceDeleteSync<Project>;
-  list is ArmResourceListByParent<Project>;
+  get is ArmResourceRead<Models.Project>;
```

Comment:
would it be simple if we just add `using Neon.Postgres.Models` to remove all the `Models.` namespace in all these places?

Reply:
Done. Changed the namespace to Neon.Postgres everywhere